src/Symfony/Component/Dotenv/Command/DebugCommand.php

```diff
@@ -47,6 +48,24 @@ public function __construct(string $kernelEnvironment, string $projectDirectory)
         parent::__construct();
     }
 
+    protected function configure(): void
+    {
+        $this
+            ->setDefinition([
+                new InputArgument('filter', InputArgument::OPTIONAL, 'The name of an environment variable or a filter.', null, $this->getAvailableVars(...)),
+            ])
+            ->setHelp(<<<'EOT'
+The <info>%command.full_name%</info> command displays all the environment variables configured by dotenv:
+
+  <info>php %command.full_name%</info>
+
+To get specific variable, specify its name:
+
+    <info>php %command.full_name% FOO_BAR</info>
+EOT
```

Comment:
We use to add a blank line at the end of the help test. Example: https://github.com/symfony/symfony/blob/v6.0.10/src/Symfony/Bundle/FrameworkBundle/Command/ContainerDebugCommand.php#L111

Reply:
added new line